src/status_im/common/floating_button_page/view.cljs

```diff
@@ -61,14 +61,17 @@
 (defn view
   [{:keys [header footer customization-color footer-container-padding header-container-style
            content-container-style gradient-cover? keyboard-should-persist-taps shell-overlay?
-           blur-options content-avoid-keyboard? automatically-adjust-keyboard-insets]
+           blur-options content-avoid-keyboard? automatically-adjust-keyboard-insets
+           ;; Note: Provide `initial-header-height` to avoid a jump due to the on-layout 1-frame
+           ;; delay. Revisit this on RN 0.76 since the on-layout delay has been fixed.
+           initial-header-height]
     :or   {footer-container-padding (safe-area/get-top)}}
    & children]
   (reagent/with-let [scroll-view-ref              (atom nil)
                      set-scroll-ref               #(reset! scroll-view-ref %)
                      window-height                (:height (rn/get-window))
                      footer-container-height      (reagent/atom 0)
-                     header-height                (reagent/atom 0)
+                     header-height                (reagent/atom nil)
                      content-container-height     (reagent/atom 0)
```

Comment:
Makes sense @ulisesmac.

Side note (not a suggestion for change): Nowadays I usually prefer to avoid nullable types if possible. In conditionals the convenience of `(when header-height ...)` won't work, that's the tradeoff, and we have to differentiate between the zero value as meaning absence with something like `(when (pos? header-height) ...)`. Similar to how in Go we avoid nil numbers, unless we the zero value is actually a valid case to consider https://go.dev/tour/basics/12. I find this practice forces me to think more about zero values and edge cases. Nullable numbers can cause problems down the road.